packages/ws-server/src/webSocketServer.ts

```diff
@@ -104,7 +102,7 @@ async function validateSubscribeEthLogsParams(filters: any, requestId: string) {
 
 app.ws.use(async (ctx) => {
     ctx.websocket.id = relay.subs()?.generateId();
-    logger.info(`New connection ${ctx.websocket.id}`);
+    logger.info(`New connection established. ConnectionId: ${ctx.websocket.id}. Current active connections: ${ctx.app.server._connections}`);
```

Comment:
do we want to log all these at `info` level?

Reply:
My reasoning is the following - all logs that indicate the occurance of an event should be `info` and all logs that contain more detailed data should be `debug` and `trace`. By changing this env `LOG_LEVEL=info` we can filter out a lot of the clutter in the logs. I found this really useful when developing and debugging localy. Why do you think that those logs should be with `debug` level?